src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/resource/AdjudicationsMappingResource.kt

```diff
@@ -73,6 +74,48 @@ class AdjudicationsMappingResource(private val mappingService: AdjudicationMappi
       )
     }
 
+  @PreAuthorize("hasRole('ROLE_NOMIS_ADJUDICATIONS')")
+  @PostMapping("/mapping/adjudications/all")
```

Comment:
have not added individual punishment and hearings yet. But will be required for synchronisation